### PR TITLE
Hubsize chart

### PR DESCRIPTION
**Pull request by ngraham20**
the hubSize value needs to be included in each of the values.yaml files.

There is one template that has not yet been updated, cluster-permission, as that values.yaml file is copied directly from the git repository https://github.com/stolostron/cluster-permission.git. This will need to be updated there before it can be updated here.

### CODE REVIEW

**Comment by dislbenn** [comment]
@ngraham20 We also need to update the `MultiClusterHub` CRD default value from `Medium` to `Small`.

**Comment by ngraham20** [comment]
Yep, I figured we'd want that to be in a separate PR, but I don't mind including it in this one

File: pkg/templates/charts/toggle/cluster-permission/values.yaml [hunk 1]
```diff
@@ -1,13 +1,11 @@
 global:
   imageOverrides:
     cluster_permission: ''
-  templateOverrides: {}
```
[the remark this answers is not in the document]
Comment (ngraham20) [reply pkg/templates/charts/toggle/cluster-permission/values.yaml:4]: I let Xiangjing know we want this value, as that's getting removed by automation

**Review by dislbenn — APPROVED** [review]
/lgtm

**Pull request merged**